src/faction.cpp

```diff
@@ -378,7 +378,7 @@ bool faction::matches_us(faction_value v) const
 std::string faction::describe() const
 {
     std::string ret;
-    ret = desc + "\n \n" + string_format( _("%1$s have the ultimate goal of %2$s."), name.c_str(),
+    ret = string_format( "%1$s\n \n%2$s have the ultimate goal of %3$s.", _( desc.c_str() ), _( name.c_str() ),
```

Comment:
Shouldn't it be `string_format( _( "%1$s\n \n%2$s have the ultimate goal of %3$s." ), ... )` ?

Also, I'd rather fix the issue like this:
```c++
std::string ret = _( desc.c_str() );
ret = ret + "\n\n" + string_format( _( "%1$s have the ultimate goal of %2$s." ), _( name.c_str() ), _( facgoal_data[goal].name.c_str() ) );
```

Reply:
I agree.